pkg/cluster/cluster.go

```diff
@@ -196,14 +259,26 @@ func (p *Peer) Peers(t PeerType) (ps []string) {
 	return ps
 }
 
+func AnyStorePeerCond() func(PeerType) bool {
+	return func(t PeerType) bool {
+		return t == PeerTypeStoreGCS || t == PeerTypeStoreSidecar
+	}
+}
+
+func PeerCond(wanted PeerType) func(PeerType) bool {
```

Comment:
so.. I thought it will be not obvious what operator we have between slice elements.. but I guess no peer can be STORE and source ;p so it makes sense to assume OR here